wrappers/C_CPP/refprop/include/Refprop/utils.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
```

Comment:
I think you are missing some STL headers in this header

Reply:
They are in the C++, but you should include what you use and not count on them being available because of previous include